src/packages/migrate/src/__tests__/DbSeed.test.ts

```diff
@@ -67,213 +103,156 @@ describe('seed', () => {
     ).toMatchInlineSnapshot(``)
   })
+})
 
-  it('one broken seed.js file', async () => {
-    ctx.fixture('edited-and-draft')
-    ctx.fs.remove('prisma/seed.js')
-    ctx.fs.remove('prisma/seed.ts')
-    ctx.fs.remove('prisma/seed.sh')
-    fs.write('prisma/seed.js', 'BROKENCODE;;;;;')
-
-    const result = DbSeed.new().parse(['--preview-feature'])
-    await expect(result).rejects.toThrowError()
-    expect(
-      ctx.mocked['console.info'].mock.calls.join('\n'),
-    ).toMatchInlineSnapshot(`Running seed from "prisma/seed.js" ...`)
-    expect(ctx.mocked['console.error'].mock.calls.join()).toMatchInlineSnapshot(
-      ``,
-    )
-  })
-
-  it('Custom --schema', async () => {
-    ctx.fixture('seed-sqlite')
+describe('seed - legacy', () => {
+  // legacy flag should warn
+  it('using --schema should warn', async () => {
+    ctx.fixture('seed-sqlite-js')
 
     const result = DbSeed.new().parse([
       '--schema=./some-folder/schema.prisma',
       '--preview-feature',
     ])
     await expect(result).resolves.toMatchInlineSnapshot(`
 
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        🌱  Your database has been seeded.
-                                                                                                                                                                                                                                                                                                                                                                                                                                    `)
+                                                                                    🌱  The seed command has been executed.
+                                                                      `)
     expect(
       ctx.mocked['console.info'].mock.calls.join('\n'),
-    ).toMatchInlineSnapshot(`Running seed from "some-folder/seed.js" ...`)
+    ).toMatchInlineSnapshot(`Running seed command \`node prisma/seed.js\` ...`)
+    expect(
+      ctx.mocked['console.warn'].mock.calls.join('\n'),
+    ).toMatchInlineSnapshot(
+      `prisma:warn The "--schema" parameter is not used anymore by "prisma db seed" since 2.27.0 and can now be removed.`,
+    )
     expect(
       ctx.mocked['console.error'].mock.calls.join('\n'),
     ).toMatchInlineSnapshot(``)
   })
 
-  it('Custom --schema from package.json', async () => {
-    ctx.fixture('seed-sqlite-schema-from-package-json')
+  it('custom --schema from package.json should enrich help setup', async () => {
```

Comment:
What does "should enrich help setup" mean? Should it influence the path that is output in the setup help output?

Reply:
Yes!